### ISSUE TITLE
Turn off parallelization across subjects, in favor of parallelization within subjects in segmentation

### ISSUE THREAD

**Issue by arokem** [yeatmanlab/pyAFQ#719]
Otherwise, a reasonable user will probably run out of memory very quickly, if they try to run this on a non-trivual number of subjects with fairly large data (e.g., a set of subjects from HCP, as I just witnessed)

**Comment by jyeatman** [comment]
parallelization across subjects is still an option right? It's just not the default?

**Comment by arokem** [comment]
Yes - to clarify: this is just an issue to remind us to adjust the default.
